libs/openchallenges/api-description/src/components/schemas/ChallengeSlug.yaml

```diff
@@ -1,4 +1,4 @@
-description: The slug of the challenge.
+description: The unique slug of the challenge.
```

Comment:
Currently the challenge slug does not have to be unique because the page of a given challenge if fetched by its ID. The motivation is to make it easier to generate slug programmatically in the future without needing have a HITL (human in the loop). For now let's keep the extra constraint and remove it only when needed.